pydoctor/model.py

```diff
@@ -78,6 +78,34 @@ class PrivacyClass(Enum):
     PRIVATE = 1
     VISIBLE = 2
 
+class _KindEnum(Enum):
+    """
+    Abstract container for L{KindClass} values. 
+    """
+    @property
+    def name(self) -> str: # type: ignore[override]
+        return super().name.replace("_", " ")
+
+class KindClass(_KindEnum):
+    """
+    Values indicating the possible object types.
+
+    @note: Some new kinds can be dynamically created by using `zope.schema`.
+        New kinds will have lower values. 
```

Comment:
It’s really the same to me, we have to sort with the minus sign and it does the trick. This is how we sort by privacy

Reply:
We should probably just use `enum.auto()`, since these values are purely internal details that aren't exported or persisted.

Deriving presentation order from the enum values at all is also an example of putting presentation details in the model, but separating it would require extra code for little practical benefit, so I don't think that's necessary.